axes/apps.py

```diff
@@ -5,7 +5,7 @@ class AppConfig(apps.AppConfig):
     name = 'axes'
 
     def ready(self):
-        from django.contrib.auth import views as auth_views
+        from django.contrib.auth.views import LoginView
         from axes.decorators import watch_login
 
-        auth_views.login = watch_login(auth_views.login)
+        LoginView.dispatch = watch_login(LoginView.dispatch)
```

Comment:
@camilonova The Debian package for django-axes 2.3.3 has a patch to fix  the tests, which corresponds to #223 

This line can be written as follows using a decorator called `method_decorator` from django.
```
LoginView.dispatch = method_decorator(watch_login)(LoginView.dispatch)
```

See the full patch here.
https://anonscm.debian.org/cgit/freedombox/django-axes.git/commit/?id=c8c8e167e3d1be4754dee04cbbe18ff35e2b57ef

With this change, the tests are passing without the refactoring from `decorated_login` to `decorated_dispatch`. Maybe that breaking api change can be avoided this way.